test/test_equivalencies.py

```diff
@@ -0,0 +1,30 @@
+"""Test different algorithms with known equivalencies against each other"""
```

Comment:
Given the theme of this file, would it be appropriate to move the "metropolis = metropolis hastings when transition probabilities/proposal distributions are symmetric" test here from the MH-specific tests? Or are those closely-enough related that they should just stay where they are?

Reply:
I like the idea of moving them here